apps/ledger-live-mobile/detox.config.js

```diff
@@ -0,0 +1,96 @@
+const androidArch = process.env.CI ? "x86" : "arm64-v8a";
+
+module.exports = {
+  testRunner: "jest",
+  runnerConfig: "e2e/config.json",
+  specs: "e2e/specs",
+  behavior: {
+    // NOTE: https://github.com/wix/Detox/blob/master/docs/APIRef.Configuration.md#behavior-configuration
+    init: {
+      reinstallApp: true,
+      exposeGlobals: true,
+    },
+    launchApp: "auto",
+    cleanup: {
+      shutdownDevice: !!process.env.CI,
+    },
+  },
+  apps: {
+    "ios.debug": {
+      type: "ios.app",
+      build: `export ENVFILE=.env.mock && xcodebuild ARCHS=${iosArch} ONLY_ACTIVE_ARCH=no -workspace ios/ledgerlivemobile.xcworkspace -scheme ledgerlivemobile -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build`,
+      binaryPath:
+        "ios/build/Build/Products/Debug-iphonesimulator/ledgerlivemobile.app",
+      // "launchArgs": "https://github.com/wix/Detox/blob/master/docs/APIRef.LaunchArgs.md",
+    },
+    "ios.staging": {
+      type: "ios.app",
+      build: `export RCT_NO_LAUNCH_PACKAGER=true && export ENVFILE=.env.mock && xcodebuild ARCHS=${iosArch} ONLY_ACTIVE_ARCH=no -workspace ios/ledgerlivemobile.xcworkspace -scheme ledgerlivemobile -configuration Staging -sdk iphonesimulator -derivedDataPath ios/build`,
+      binaryPath:
+        "ios/build/Build/Products/Staging-iphonesimulator/ledgerlivemobile.app",
+    },
+    "ios.release": {
+      type: "ios.app",
+      build: `export RCT_NO_LAUNCH_PACKAGER=true && export ENVFILE=.env.mock && xcodebuild ARCHS=${iosArch} ONLY_ACTIVE_ARCH=no -workspace ios/ledgerlivemobile.xcworkspace -scheme ledgerlivemobile -configuration Release -sdk iphonesimulator -derivedDataPath ios/build`,
+      binaryPath:
+        "ios/build/Build/Products/Release-iphonesimulator/ledgerlivemobile.app",
+    },
+    "android.debug": {
+      type: "android.apk",
+      build:
+        "cd android && ENVFILE=.env.mock ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ..",
+      binaryPath: `android/app/build/outputs/apk/debug/app-${androidArch}-debug.apk`,
+      testBinaryPath:
+        "android/app/build/outputs/apk/androidTest/debug/app-debug-androidTest.apk",
+    },
+    "android.release": {
+      type: "android.apk",
+      build:
+        "cd android && ENVFILE=.env.mock ./gradlew assembleStagingRelease assembleAndroidTest -DtestBuildType=stagingRelease && cd ..",
+      binaryPath: `android/app/build/outputs/apk/stagingRelease/app-${androidArch}-stagingRelease.apk`,
+      testBinaryPath:
+        "android/app/build/outputs/apk/androidTest/stagingRelease/app-stagingRelease-androidTest.apk",
+    },
+  },
+  devices: {
+    simulator: {
+      type: "ios.simulator",
+      device: {
+        type: "iPhone 11 Pro",
+      },
+      // bootArgs: "https://github.com/wix/Detox/blob/master/docs/APIRef.Configuration.md",
+    },
+    emulator: {
+      type: "android.emulator",
+      device: {
+        avdName: "Nexus_6",
```

Comment:
nexus6 might be a bit outdated if we ever want to support android for this

Reply:
I don't think the virtual device matters too much aside from the screen size and resolution, the main things are the emulator architecture, the API level and the target (google APIs, AOSP, default). 

The actual command used looks like this: `/bin/sh -c \echo no | avdmanager create avd --force -n Nexus_6 --abi 'google_apis/x86' --package 'system-images;android-28;google_apis;x86'`, so it's just currently a name and we could change it to anything (so we should actually change it to reflect that). If we wanted to actually use a specific device emulator we would pass in `--device "pixel"`